benchmarks/opmphm.c

```diff
@@ -25,6 +25,21 @@
 #endif
 #include <sys/time.h>
 
+// mingw hustle
+#include <inttypes.h>
+#ifdef _WIN32
+#	ifdef _WIN64
+#		define PRINT_SIZE_T PRIu64
```

Comment:
If you want to, you can also add this to `src/include/kdbtypes.h` but then it would need a `ELEKTRA_` prefix and `_F` suffix.

Reply:
I thought of that, but `inttypes.h` gets included in `kdbos.h` only if `#ifndef _WIN32` that confused me.

Reply:
Well I need to find another solution this macro does not work, now i get:
```
 warning: ISO C does not support the ‘I64’ ms_printf length modifier [-Wformat=]
```

Reply:
> inttypes.h gets included in kdbos.h only if #ifndef _WIN32

I assume that inttypes.h was not present on _WIN32 but you are welcomed to retest if this is still the case.

> Well I need to find another solution this macro does not work, now i get:

Yes, the C99 support is rather limited.

We already printf size_t at plenty other places and I would assume that some of them are already built with mingw?

Reply:
I will exclude the benchmark for mingw, it is not worth the effort.